.pre-commit-config.yaml

```diff
@@ -12,6 +12,14 @@ repos:
       - id: check-toml
       - id: check-json
 
+  - repo: https://github.com/python-poetry/poetry
+    rev: '1.8.3'
+    hooks:
+      - id: poetry-check
```

Comment:
this will fail if the lockfile hasn't been updated, or if there's a typo in a package name in poetry config.